proto/src/rr/rdata/aaaa.rs

```diff
@@ -39,16 +39,15 @@ use error::*;
 
 /// Read the RData from the given Decoder
 pub fn read(decoder: &mut BinDecoder) -> ProtoResult<Ipv6Addr> {
-    let a: u16 = try!(decoder.read_u16());
-    let b: u16 = try!(decoder.read_u16());
-    let c: u16 = try!(decoder.read_u16());
-    let d: u16 = try!(decoder.read_u16());
-    let e: u16 = try!(decoder.read_u16());
-    let f: u16 = try!(decoder.read_u16());
-    let g: u16 = try!(decoder.read_u16());
-    let h: u16 = try!(decoder.read_u16());
-
-    Ok(Ipv6Addr::new(a, b, c, d, e, f, g, h))
+    Ok(Ipv6Addr::new(
```

Comment:
I'm 50/50 on this improving readability...

Reply:
yeah I wasn't sure either about this one.